hudi-hadoop-common/src/test/java/org/apache/hudi/common/functional/TestHoodieLogFormat.java

```diff
@@ -2780,27 +2773,6 @@ public void testGetRecordPositions(boolean addRecordPositionsHeader) throws IOEx
     }
   }
 
-  public static HoodieDataBlock getDataBlock(HoodieLogBlockType dataBlockType, List<IndexedRecord> records,
```

Comment:
The method here and below are moved to `HoodieCommonTestHarness` for better reusability.